app/controllers/api/v1/packages_controller.rb

```diff
@@ -130,6 +130,7 @@ def search_stockit_items
         records = records.search(search_text: params['searchText'], item_id: params['itemId'],
           with_inventory_no: params['withInventoryNumber'] == 'true') if params['searchText'].present?
         params_for_filter = ['state', 'location'].each_with_object({}){|k, h| h[k] = params[k] if params[k].present?}
+        records =  records.inventorize_and_not_multiquantity if params["isOrdersSearch"]
```

Comment:
If think the param name should be more explicit as to what it does, `isOrdersSearch` does not give any information as to what it does. I'd argue towards something explicit like `?allowMultiQuantity=true|false` (example)